sigopt_example.m

```diff
@@ -1,40 +1,43 @@
-% 
-%
-% define dimensions
-%
-% make connection object
-conn = py.sigopt_matlab.get_connection()
+import sigopt.Connection
 
-% dimension 1
-x_bounds = py.dict(pyargs('min', 0, 'max', 1))
-x_parameter = py.dict(pyargs('name', 'x', ...
-                             'type', 'double', ...
-                             'bounds', x_bounds))
-                        
-% dimension 2                
-y_bounds = py.dict(pyargs('min', 0, 'max', 1))
-y_parameter = py.dict(pyargs('name', 'y', ...
-                             'type', 'double', ...
-                             'bounds', y_bounds))
+conn = Connection('<YOUR API TOKEN HERE>');
 
-                    
-% define parameters and name variables
-parameters = py.list({x_parameter, y_parameter})
-name = 'Franke Optimization (MATLAB)'
+experiment = conn.experiments().create(struct( ...
+  'name', 'Franke Optimization (MATLAB)', ...
+  'parameters', [ ...
+    struct( ...
+      'name', 'x', ...
+      'type', 'double', ...
+      'bounds', struct( ...
+        'min', 0, ...
+        'max', 1 ...
+      ) ...
+    ), ...
+    struct( ...
+      'name', 'y', ...
+      'type', 'double', ...
+      'bounds', struct( ...
+        'min', 0, ...
+        'max', 1 ...
+      ) ...
+    ) ...
+  ] ...
+))
 
-% make experiment
-experiment = py.sigopt_matlab.make_new_experiment(name, parameters, conn)
-experiment_id = py.getattr(experiment, 'id')
+experiment_id = experiment.id;
 
 % optimize!
```

Comment:
I like to use very specific wording, for example, calling the lop "The optimization loop" and saying "receive a suggestion, evaluate the metric, report an observation" for the three steps